src/elements/Select.tsx

```diff
@@ -18,6 +18,7 @@ export interface SelectProps extends PositionProps, SpaceProps {
   options: Option[]
   selected?: string
   disabled?: boolean
+  error?: string
```

Comment:
Should `error` be a boolean?

Reply:
If error will eventually display message it might be clearer to label this prop `errorMsg` and then check against that, with a little `TODO: Implement ErrorMessage component` somewhere.

Reply:
happy to adapt this however is good- but in reaction Input takes an ` error?: string`. Your thoughts?

Reply:
Ahh cool cool, this seems fine for now 👍 It would be good to add a `TODO` though so its clear that we haven't yet hooked up the message part, which will eventually display the string that's passed into this prop.